### DISCUSSION TITLE
Session based / Telnet console.

### DISCUSSION THREAD

**Discussion by 0xStuart** [spectreconsole/spectre.console#836] [Q&A] [answered]
Is it possible to create multiple instances of Spectre in the same app?

If you have long running server apps, without a UI, it's handy to have a console you can connect to.  A listener socket you can Telnet into which creates a console session. I find it useful for debugging, making live configuration changes, or capturing the log output in real-time.

It would be nice to make it look better with Spectre.  I want to be able to launch new instances, on demand, but couldn't see any solution to this.

**Comment by patriksvensson** [comment, marked as answer]
You can create a new `AnsiConsole` via the [AnsiConsole.Create](https://spectreconsole.net/api/spectre.console/ansiconsole/de95d17b). If you don't want to output all the consoles to stdout, you will need to implement an `IAnsiConsoleOutput` and use that one when creating the console.

**Reply by 0xStuart** [reply]
Thanks @patriksvensson It works great.  I managed to hack something together this weekend.

For those interested, https://gitlab.com/logicethos/consoleserver
Needs cleaning up a bit, and testing with more telnet clients.  I will create a Nuget at some point.
